blog/2024-11-28-developer-portal-restructure.md

```diff
@@ -0,0 +1,42 @@
+
+In this blogpost we'll quickly go over all the new releases, and how the new structure is implemented to not only make it more organized, but also to be more future-proof for new sections that will be added in the future.
+
+<!--truncate-->
+
+The Developer Portal Docs section has always consisted of a split between Guides & Tutorials and References. But as new sections have been added to the Developer Portal, such as the UI Library and Mobile UI docs, the navigation has become a bit cluttered. Therefore we decided to restructure the navigation of the Developer Portal.
+
+### UI Library
+
+The UI library used to be hosted on a separate site, but we've now moved it to the Developer Portal. The UI Library is a collection of reusable components that can be used to build your own DHIS2 applications. But not only does it contain Web UI Components, it also contains best practices, predefined colors, an icon library and other Paterns and Principles. The Design System is now separated from the Web UI components. The Design system is applicable to all DHIS2 applications (Web & Android), while the Web UI components are specifically for web applications.
+
+You can find the [Design System at the top level navigation](/design-system) of the Developer Portal, while the Web UI components are found in the [References section](/docs/ui/webcomponents).
+
+We've also moved the interactive demo into the developer portal, but it still lives as a separate page. You can find the [Interactive Demo](pathname:///demo/) from within each Web UI component page through the in-line demo's.
+
+### Mobile Documentation
+
+The Mobile Documentation has been introduced into the Developer portal only recently, and has shortly lived as a top-level navigation item. But as it *actually* had a place as a sub-section of the Guides & Tutorials section, we've moved it there. You can find the [Mobile Documentation in the Guides & Tutorials section](/docs/mobile). You can expect more content to be added to this section in the near future.
+
+### Splitting of Guides & Tutorials and References
+
+Previously both Guides & Tutorial and References were found in the same section, hidden under the "docs" label in the top-level navigation. However, as the Developer Portal has grown, we've decided to split these two parts into two separate top-level sections. Guides & Tutorials can be found in the [Guides & Tutorials section](/docs), while References can be found in the [References section](/docs/references). This makes it easier to find the information you need, and also makes it easier to add new sections in the future.
+
+## Searching within Developer Portal & AI
+
+Because of all the new sections added to the Developer Portal, the search bar on the top-right has also become more useful. Not only is it able to search whatever was there before the merge, but now it knows of many more pages. This is incredibly useful as a on-stop search.
```

Comment:
Small typo
```suggestion
Because of all the new sections added to the Developer Portal, the search bar on the top-right has also become more useful. Not only is it able to search whatever was there before the merge, but now it knows of many more pages. This is incredibly useful as a one-stop search.
```